src/templates/customer/edit_account/template.html

```diff
@@ -268,45 +261,45 @@ <h2 name="billing" id="billing">Billing Information</h2>
 			<div class="col-12 col-md-6">
 				<h2 name="contact" id="contact">Contact Information</h2>
 				<div class="form-group">
-					<label for="email">Email Address</label>
-					<span class="small text-danger">Required</span>
+					<label for="email">Email address</label><span class="text-danger">*</span>
 					<input class="form-control" type="text" name="email" id="email" size="50" value="[@email@]">
 				</div>
 				<div class="form-group">
 					<label>
 						<input type='checkbox' id="subscribe_newsletter" [%if [@opt_in_newsletter@]%]checked[%/if%]>
 						<input type="hidden" name="opt_in" value="[@opt_in_newsletter@]">
-						Subscribe To Newsletter
+						Subscribe to newsletter
 					</label>
 				</div>
 				<div class="form-group">
-					<label for="phone">Phone Number </label>
-					<span class="small text-danger">Required</span>
+					<label for="phone">Phone number</label><span class="text-danger">*</span>
 					<input class="form-control" type="text" name="phone" id="phone" size="50" value="[@phone@]">
 				</div>
 				<div class="form-group">
-					<label for="fax">Fax Number</label>
+					<label for="fax">Fax number</label>
 					<input class="form-control" type="text" name="fax" id="fax" size="50" value="[@fax@]">
 				</div>
 				<!-- Company Information -->
 				<h2 name="company" id="Company">Company Information</h2>
+				[%if [@config:defaultcountry@] eq 'AU'%]
```

Comment:
As discussed these ABN configs are kinda of a small mess.

Perhaps we change this logic to 
`If defaultcountry == 'AU' || primary_abn_label != 'ABN'`
??? 😩